src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -9,6 +9,11 @@ CA1841 | Performance | Info | PreferDictionaryContainsMethods, [Documentation](h
 CA1842 | Performance | Info | DoNotUseWhenAllOrWaitAllWithSingleArgument, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1842)
 CA1843 | Performance | Info | DoNotUseWhenAllOrWaitAllWithSingleArgument, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1843)
 
+### New Rules
+Rule ID | Category | Severity | Notes
```

Comment:
Recently enabled markdownlint for analyzer releases files in `main` branch. This avoids failure later on.

```suggestion
### New Rules

Rule ID | Category | Severity | Notes
```